model/src/ww3_shel.F90

```diff
@@ -600,6 +600,9 @@ PROGRAM W3SHEL
   NDS(11) = 22
   NDS(12) = 23
   NDS(13) = 34
+  NDS(14) = 36
```

Comment:
I've used unit numbers 36 and 37 for these new outputs - as far as I can tell they are not used elsewhere.

Reply:
I grep'ed around and could not find any instances of 36 or 37 being used for unit numbers.